packages/maps/src/lib/mapControlGroup/MapControlGroup.svelte

```diff
@@ -12,7 +12,7 @@
 	}
 
 	type PositionClass = {
-		[key: MapControlGroupPositions]: string;
+		[key: string]: string;
```

Comment:
```suggestion
		[key in keyof typeof MapControlGroupPositions]: string;
```

This has the effect of defining`PositionClass` to be:

```ts
type PositionClass = {
    readonly TopLeft: string;
    readonly TopCenter: string;
    readonly TopRight: string;
    readonly TopRightOffset: string;
    readonly CenterRight: string;
    readonly BottomRight: string;
    readonly BottomCenter: string;
    readonly BottomLeft: string;
    readonly CenterLeft: string;
}
```